aiida/orm/nodes/node.py

```diff
@@ -703,6 +761,9 @@ def put_object_from_file(self, path, key, mode=None, encoding=None, force=False)
         # order to detect when they were being passed such that the deprecation warning can be emitted. The defaults did
         # not make sense and so ignoring them is justified, since the side-effect of this function, a file being copied,
         # will continue working the same.
+        if force:
+            warnings.warn('the `force` keyword is deprecated and will be removed.', AiidaDeprecationWarning)  # pylint: disable=no-member
```

Comment:
We should probably mention in the deprecation warning that it no longer has any effect.

Reply:
Actually, looking at the code, it _does_ currently still have effect. Sure, when it will be removed it no longer has an effect, but I think that would be obvious no :sweat_smile:  Is there something else you meant?

Reply:
Ah, yes. I was probably thinking about the branch where the new repository is already implemented.